src/browserbehavior/BrowserBehavior.ts

```diff
@@ -27,6 +27,11 @@ export default interface BrowserBehavior {
    */
   hasFirefoxWebRTC(): boolean;
 
+  /**
+   * Returns whether the browser requires the "playback" latency hint for Web Audio
+   */
+  requiresPlaybackLatencyHintForAudioContext(): boolean;
```

Comment:
If wont make this optional this would be a breaking change right? Not sure if builders are using this interface but to consider if they are using it this would be a breaking change I believe.

Tagging @justindarc @ltrung